### PR TITLE
Test full code in real-world setting and make bug fixes.

### PR DESCRIPTION
**Pull request by KathyFeiyang**
### What is a quick description of the change?

**Test the full code of voter-central ([web application](https://github.com/googleinterns/voter-central/tree/master/project) + [data collection script](https://github.com/googleinterns/voter-central/tree/master/info-compiler)) in real-world setting and make bug fixes, along with commits:**
- https://github.com/googleinterns/voter-central/pull/27/commits/bba1cf5e9f93704ddba8cb00d2b4da74455ff67d
- https://github.com/googleinterns/voter-central/pull/27/commits/8d689f3a64790a5f431ac58fc40c291983b413a3

### Is this fixing an issue?

none

### Are there more details that are relevant?

none

### Check lists (check `x` in `[ ]` of list items)

- [x] Test written/updating
- [x] Tests passing
- [x] Coding style (indentation, etc)

Please explain why any are not present, if any.

### CODE REVIEW

File: info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java [hunk 1]
```diff
@@ -187,13 +187,13 @@ public void resetDatastore() throws IOException {
...
-    assertThat(infoCompiler.addresses).hasSize(ADDRESS_NUMBER);
-    assertThat(infoCompiler.addresses).contains(ADDRESS);
-  }
+  // @Test
+  // public void parseAddressesFromDataset_regularParse() {
+  //   // The list of U.S. addresses in the dataset should contains {@code ADDRESS_NUMBER} addresses
+  //   // and contain {@code ADDRESS}.
+  //   assertThat(infoCompiler.addresses).hasSize(ADDRESS_NUMBER);
+  //   assertThat(infoCompiler.addresses).contains(ADDRESS);
+  // }
```
Comment (kellyktan) [inline info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java]: nit: don't include commented tests.

File: info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java [hunk 2]
```diff
@@ -127,7 +128,7 @@ public void compileNewsArticle(String candidateName, String candidateId, String
     }
   }
 
-  // @TODO [Might adopt in {@code compileNewsArticle}.]
+  // [Might adopt in {@code compileNewsArticle}.]
```
Comment (kellyktan) [inline info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java:131]: Is this not a TODO anymore?

**Review by kellyktan — APPROVED** [review]

File: info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java (hunk 2 above)
Comment (KathyFeiyang) [reply info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java:131]: My original intention was to indicate that the implementation is not a TODO anymore. It's just that we are currently not using this method. Should I add back the TODO?

File: project/src/main/java/com/google/sps/servlets/CandidateServlet.java [hunk 3]
```diff
@@ -98,7 +98,7 @@ private Candidate getCandidateData(String candidateId, String electionName) {
         (String) candidateData.getProperty("party"),
         (String) candidateData.getProperty("email"),
         (String) candidateData.getProperty("phone"),
-        (String) candidateData.getProperty("PhotoUrl"),
+        (String) candidateData.getProperty("photoUrl"),
```
Comment (suparnabarua) [inline project/src/main/java/com/google/sps/servlets/CandidateServlet.java:101]: I thought @tjyoung99  mentioned the API returned this as "PhotoUrl"?

**Review by suparnabarua — CHANGES_REQUESTED** [review]

File: project/src/main/java/com/google/sps/servlets/CandidateServlet.java (hunk 3 above)
Comment (KathyFeiyang) [reply project/src/main/java/com/google/sps/servlets/CandidateServlet.java:101]: Yes. InfoCompiler currently stores it as "photoUrl" and the backend & frontend uses it as "photoUrl" in the merged version. However, I can change everything to "PhotoUrl" as well

Comment (KathyFeiyang) [reply project/src/main/java/com/google/sps/servlets/CandidateServlet.java:101]: Actually, I checked the API: https://developers.google.com/civic-information/docs/v2/elections/voterInfoQuery. The API returns this as "photoUrl"

File: info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java (hunk 2 above)
Comment (kellyktan) [reply info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java:131]: Oh actually can you just remove the whole commented out method in that case?

File: project/src/main/java/com/google/sps/servlets/CandidateServlet.java (hunk 3 above)
Comment (suparnabarua) [reply project/src/main/java/com/google/sps/servlets/CandidateServlet.java:101]: Thanks for confirming

**Review by suparnabarua — APPROVED** [review]

File: info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java (hunk 2 above)
Comment (KathyFeiyang) [reply info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java:131]: Gotcha!

File: project/src/main/java/com/google/sps/servlets/CandidateServlet.java (hunk 3 above)
Comment (KathyFeiyang) [reply project/src/main/java/com/google/sps/servlets/CandidateServlet.java:101]: No problem!

**Review by ganewman — APPROVED** [review]
LGTM!

**Review by tjyoung99 — APPROVED** [review]

**Pull request merged**